utils/appliance/__init__.py

```diff
@@ -2607,11 +2608,19 @@ def get_or_create_current_appliance():
 current_appliance = LocalProxy(get_or_create_current_appliance)
 
 
+@removals.removed_class(
+    "CurrentAppliance", message=("The CurrentAppliance descriptor is being phased out"
+                                 "in favour of collections.")
+)
 class CurrentAppliance(object):
     def __get__(self, instance, owner):
         return get_or_create_current_appliance()
 
 
+@removals.removed_class(
+    "Navigatable", message=("Navigatable is being deprecated in favour of using Collections "
+                            "objects with the NavigatableMixin")
+)
 class Navigatable(object):
```

Comment:
well done, for avoiding duplication it may help to make `Navigatable` a subclass of `NavigatableMixin` just like sentaku does with `Element` vs `ElementMixin`

Reply:
Yeh I thought of that, will do it